fileshare/event_manager_test.go

```diff
@@ -41,12 +41,35 @@ type mockNotification struct {
 type mockNotifier struct {
 	notifications []mockNotification
 	nextID        uint32
+	updateCh      chan struct{}
+}
+
+func (mn *mockNotifier) Wait() {
+	timeout := make(chan bool, 1)
+	go func() {
+		time.Sleep(1 * time.Second)
+		timeout <- true
+	}()
+
+	select {
+	case <-timeout:
```

Comment:
Can we use `<-time.After(1 * time.Second)` here? I think you won't need to spawn separate goroutine then